source/reference/minio-mc-admin/mc-admin-user-add.rst

```diff
@@ -0,0 +1,130 @@
+.. _minio-mc-admin-user-add:
+
+=====================
+``mc admin user add``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin user add
+
+
+Syntax
+------
+
+.. start-mc-admin-user-add-desc
+
+The :mc:`mc admin user add` command adds a new user to the target MinIO deployment.
```

Comment:
```suggestion
The :mc:`mc admin user add` command adds a new :ref:`MinIO user <minio-internal-idp>` to the target MinIO deployment.
```

Reply:
I like this idea 👍🏻

Reply:
Added additional links to the subcommand pages, except `svcacct`. Service accounts could also be LDAP. Also slightly different text for `sts info` because context.

Reply:
@ravindk89 restaged if you want to have a look in situ before I declare this done.
http://192.241.195.202:9000/staging/DOCS-785/linux/html/reference/minio-mc-admin/mc-admin-user-add.html
http://192.241.195.202:9000/staging/DOCS-785/linux/html/reference/minio-mc-admin/mc-admin-user-list.html